Samples/src/Arithmetic/Adder Example.ipynb

```diff
@@ -0,0 +1,232 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Integer arithmetic\n",
+    "\n",
+    "Many quantum algorithms need to evaluate classical functions on a superposition of inputs. An example is Shor's algorithm, which requires the mapping $|x\\rangle|0\\rangle\\mapsto|x\\rangle|a^x\\text{ mod }N\\rangle$ to be implemented as a quantum operation.\n",
+    "\n",
+    "To keep users from having to implement such functions from scratch, the Canon (`Microsoft.Quantum.Canon`) contains arithmetic functionality such as integer adders."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's write a wrapper for one of these adders. This will allow us to quickly exchange it for a different implementation if need be."
```

Comment:
```suggestion
    "Let's write a wrapper for one of these adders, that of[Takahashi *et al*](https://arxiv.org/abs/0910.2530). This will allow us to quickly exchange it for a different implementation if need be."
```

Reply:
Done